conf/messages.cy

```diff
@@ -152,11 +152,12 @@ ssttp.call-us.p2.telephone = 0300 200 1900.
 ssttp.call-us.before-call.heading = Cyn i chi ffonio, sicrhewch fod gennych y canlynol:
 ssttp.call-us.before-call.l1 = gwybodaeth am unrhyw gynilion neu fuddsoddiadau sydd gennych
 ssttp.call-us.before-call.l2 = eich manylion banc
+ssttp.call-us.before-call.l3 = manylion incwm a gwariant
 ssttp.call-us.likely-to-ask.heading = Rydym yn debygol o ofyn:
 ssttp.call-us.likely-to-ask.l1 = beth rydych wedi’i wneud i geisio talu’r bil
 ssttp.call-us.likely-to-ask.l2 = a allwch dalu rhywfaint o’r bil nawr
 ssttp.call-us.opening-times.heading = Ein horiau agor yw:
-ssttp.call-us.opening-times.l1 = Dydd Llun i ddydd Gwener: 08:30 – 16:00
+ssttp.call-us.opening-times.l1 = Dydd Llun i ddydd Gwener: 8:00 i 18:00
```

Comment:
Is the `i` in `8:00 i 18:00` correct? or should it be a '-' as it was before the change?  
(not just this change but also other time changes)

Reply:
i is the welsh translation for to

Reply:
I get to learn something today :-)